src/games/strategy/triplea/delegate/AirBattle.java

```diff
@@ -761,14 +762,8 @@ private void notifyCasualties(final GUID battleID, final IDelegateBridge bridge,
       public void run() {
         try {
           getRemote(firingPlayer, bridge).confirmEnemyCasualties(battleID, "Press space to continue", hitPlayer);
-        } catch (final ConnectionLostException cle) {
-          // somone else will deal with this
-          // System.out.println(cle.getMessage());
-          // cle.printStackTrace(System.out);
-        } catch (final GameOverException e) {
-          // ignore
         } catch (final Exception e) {
-          // ignore
+          ClientLogger.logQuietly(e);
```

Comment:
My point was there were many others exceptions that were doing nothing that you didn't add the log statement to but for some reason just this one. My point is if you start making functional changes its no longer a pure refactoring PR and needs a closer inspection. Also don't understand why this particular exception needed a log but many others didn't.